brainrender/actors/line.py

```diff
@@ -0,0 +1,22 @@
+from vedo import shapes
+
+from brainrender.actor import Actor
+
+
+class Line(Actor):
+    def __init__(
+        self, coordinates, color="black", alpha=1, linewidth=2, name=None
+    ):
+        """
+        Creates an actor representing a single line.
+
+        :param coordinates: list, np.ndarray with shape (N, 3) of ap, dv, ml coordinates.
+        :param color: str
+        :param alpha: float
+        :param linewidth: float
+        :param name: str
```

Comment:
Could you add a short description (similar to what you did for `coordinates`) for the rest of the parameters? It would be nice to provide a bit more context on what values are acceptable - for example, does the colour string only take names like "blue" or can it accept hex codes too?

Reply:
> for example, does the colour string only take names like "blue" or can it accept hex codes too

While I agree in principle that the doc-strings should contain that information, do note that **all** other actors lack this level of detail. The doc-strings for the vedo objects (which are being handed the color specifications by brainrender) also don't say, so the only way to figure this out is to run a bunch of tests. Based on my tests, CSS-color-4 named colors & hex codes are supported, as are (R, G, B) tuples. However, it is quite possible that the answer is backend-specific. As I have only been able to get the VTK backend working, I can't answer this point definitively.

Reply:
Could you add some examples, like:
```
:param color: e.g. "blue", "E7900A", (10, 230, 190)
```

We'd like the existing docstrings to not be the baseline, as they need considerable improvement. However, this is certainly not your job to improve them @paulbrodersen! Happy to leave this as is if you don't think it can be specified further.

Reply:
Addressed in 66542fd.